Add entries for replaygain update functionality. (#417)
mixxxdj/manual: source/chapters/appendix/mixxx_controls.rst

   .. versionadded:: 2.0.0

.. mixxx:control:: [ChannelN],update_replaygain_from_pregain
                   [PreviewDeckN],update_replaygain_from_pregain
                   [SamplerN],update_replaygain_from_pregain

  Applies the deck pregain knob value to the detected ReplayGain value for the
  current track. This is a way to update the ReplayGain value of a track if it

>   has been detected incorrectly. When this option is selected, the pregain value
>   for the deck will be adjusted so that there is no audible difference in track
>   volume, so this operation is safe to use during performance.

Reviewer: ```suggestion
  has been detected incorrectly. When this control is triggered, the pregain value
  for the deck will be centered so that there is no audible difference in track
  volume, so this operation is safe to use during performance.
```

Author: this is not quite true -- the deck that was the source of the adjustment will end up at the center position, but if there are any other decks with the track loaded they may not be adjusted to the center.

Author: But for the sake of simplicity, maybe it's fine to elide that detail :)

Author: applied manually

Reviewer: ah, good point.

>   track volume, so this operation is safe to use during performance.

Reviewer 2: ```suggestion
  track volume, so this operation is safe to use during performance, if the controller mapping uses soft-takeover for the pregain knob.
```

Reviewer 3: Hmm, this always applies. Otherwise even adjusting the gain knob via mouse will lead to issues, regardless of replay gain. So I IMHO this addition is not really needed.

Reviewer: This is a long sentence already. Also, a controller is not necessarily the default.
What about
"If you use a controller make sure to enable SoftTakeover for the pregain knob for full comfort"

Reviewer: > Hmm, this always applies. Otherwise even adjusting the gain knob via mouse will lead to issues, regardless of replay gain. So I IMHO this addition is not really needed.

true. let's leave it as is.

Reviewer 2: This is the documentation page for developers of controller mappings. And this is the point, which needs to be considered in a mapping, because all other COs can be used safely without implemnting soft-takeover, but this one not.

Author: I think it's fine to add this clarification

   :range: binary

>    :feedback: ReplayGain value is updated in library, deck pregain is adjusted.

Reviewer: ```suggestion
   :feedback: ReplayGain value is updated in library, deck pregain is centered.
```

Author: applied manually

Reviewer: ah okay, I just edited to "...deck pregain is reset to 1.0"

Author: done

Reviewer: thx!

>    :range: binary
>    :feedback: ReplayGain value is updated in library, deck pregain is reset to 1.0.

Reviewer: "One more thing..." ; )
```suggestion
  :range: binary
  :feedback: ReplayGain value is updated in library, deck pregain is reset to 1.0.
```
to align it with the above paragraph, like other controls
